.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
     rev: v2.2.6
     hooks:
       - id: codespell
-        args: [--write-changes]
+        exclude: ".lock$"
```

Comment:
I think it's more robust to configure this in `pyproject.toml` so it's used when `codespell` is run outside of `pre-commit`, and I did try and configure it already:
https://github.com/UCL-ARC/python-tooling/blob/d96dd24fbd8dfe719757b3e19dae79a36ae9086a/pyproject.toml#L1-L2

Is the existing config in `pyproject.toml` not working?

Reply:
I hadn't spotted that. I'm assuming that wasn't there at the start? I agree that is better.

If you read my original comments here https://github.com/UCL-ARC/python-tooling/pull/207#issue-1961407706 it clearly didn't work at some point. It is possible that that PR got branched off `main` and that is how the change to the lockfile happened. I don't know, and I couldn't replicate it.